uber/site_sections/registration.py

```diff
@@ -153,6 +153,65 @@ def change_badge(self, session, id, message='', **params):
             'attendee': attendee
         }
 
+    def qrcode_generator(self, data):
+        """
+        Takes a piece of data and encrypts it using the QR_CODE_PASSWORD
+        Then, adds the EVENT_QR_ID, and returns an Aztec barcode as an image stream.
+        Args:
+            data: A string to encrypt.
+
+        Returns: A PNG buffer. Use this function in an img tag's src='' to display an image.
+
+        """
```

Comment:
Since we're returning a png we should set the content type header here:

```python
cherrypy.response.headers['Content-Type'] = 'image/png'
```

Browsers *should* be able to look at the first few bytes of the image we return to figure out that it's a png, but this would help make it more guaranteed.

For that matter, one feature of CherryPy we seldom use but which we could use here is the fact that if someone makes an HTTP request for ``/uber/registration/qrcode_generator.png`` then CherryPy will replace the ``.`` with ``_`` when trying to find the page handler, so if we name this method ``qrcode_generator_png`` then we'd be able to write code like ``<img src="{{ c.URL_BASE }}/registration/qrcode_generator.png?data={{ attendee.id }}" />`` but that's probably overkill since literally every browser should handle a img src without an extension.